src/common/header/Header.module.css

```diff
@@ -2,10 +2,23 @@
   align-items: center;
   border-bottom: 2px solid black;
   display: flex;
-  padding: 10px;
+  padding: 10px 0;
 }
 
-.appName {
-  font-size: 24px;
-  margin-left: 30px;
-}
\ No newline at end of file
+
+  .appName {
```

Comment:
Make these styles mobile-first. To do so, copy the styles for the **smallest** screen here (mobile-first!), then start applying styles that kick into action **after** a certain width, avoid max-width media queries. Like this:

Mobile:
```
.appName {
  font-size: 16px;
  margin-left: 0;
}
```

Next smallest screen:
```
@media (min-width: 415px) {
  .appName {
    font-size: 24px;
    font-weight: bold;
    margin-left: 10px;
  }
}
```